client/app/domain/webhosting-enable/domain-enable-web-hosting.html

```diff
@@ -46,22 +46,21 @@
                     </thead>
                     <tbody>
                         <tr data-ng-repeat="offer in ctrl.offers | orderBy:orderByOfferPrice track by $index">
-                            <td>
+                            <td colspan="2">
                                 <div class="oui-radio m-0">
                                     <input type="radio" class="oui-radio__input" id="offer-{{offer.offer}}" name="hostingOffer"
                                            data-ng-disabled="offer.offer === ctrl.constants.HOSTING.OFFERS.START_10_M.LIST_VALUE && ctrl.model.templateSelected"
                                            data-ng-model="ctrl.model.offer"
                                            data-ng-value="offer.offer">
-                                    <span class="oui-radio__label-container"></span>
+                                    <label class="oui-radio__label-container" for="offer-{{offer.offer}}">
```

Comment:
No `id` corresponds to `label-{{ offer.offer }}`, is there ?

Reply:
@FredericEspiau -> `id="offer-{{offer.offer}}"` line 51 :)

Reply:
My bad !